.github/workflows/master.yaml

```diff
@@ -52,30 +63,30 @@ jobs:
           gcloud --quiet auth configure-docker
       - name: Build controller
         run: |
-          make docker-build
-          make docker-push
-      - name: Build tool
-        uses: goreleaser/goreleaser-action@v1
-        with:
-          args: release --rm-dist --release-notes hack/changelog.md.txt
-          key: ${{ secrets.BMASTERS_PRIVATE_KEY }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.BMASTERS_TOKEN }}
-      - name: Tag Docker images
-        run: |
-          docker tag "${IMG}" "${IMG%%:*}:${DOCKER_TAG}"
-          docker push "${IMG%%:*}:${DOCKER_TAG}"
-          docker tag "${REDSKYCTL_IMG}" "${REDSKYCTL_IMG%%:*}:${DOCKER_TAG}"
-          docker push "${REDSKYCTL_IMG%%:*}:${DOCKER_TAG}"
-          docker tag "${SETUPTOOLS_IMG}" "${SETUPTOOLS_IMG%%:*}:${DOCKER_TAG}"
-          docker push "${SETUPTOOLS_IMG%%:*}:${DOCKER_TAG}"
-      - name: Upload macOS binary
-        uses: actions/upload-artifact@v1
-        with:
-          name: redskyctl_darwin_amd64
-          path: dist/redskyctl-darwin-amd64.tar.gz
-      - name: Upload Linux binary
-        uses: actions/upload-artifact@v1
-        with:
-          name: redskyctl_linux_amd64
-          path: dist/redskyctl-linux-amd64.tar.gz
+          make docker-build-ci
+#          make docker-push
```

Comment:
Will uncomment these steps after a success. Such a pain to test these workflows :/